checkST2110.js

```diff
@@ -473,35 +577,54 @@ const extractMTParams = (sdp, params = {}) => {
   let isSkippedType = false;
   let streamCount = 0;
   let payloadType = -1;
-  for ( let x = 0 ; x < lines.length ; x++ ) {
+  let mediaType = -1;
+  let encodingName = -1;
+
+  for (let x = 0; x < lines.length; x++) {
     if (lines[x].startsWith('m=')) {
       let videoMatch = lines[x].match(videoPattern);
-      payloadType = videoMatch ? +videoMatch[4] : -1;
+      let audioMatch = lines[x].match(audioPattern);
+      if (videoMatch) {
+        mediaType = 'video';
+        payloadType = +videoMatch[4];
+      }
+      else if (audioMatch) {
+        mediaType = 'audio';
+        payloadType = +audioMatch[4];
+      }
+      else {
+        errors.push(new Error(`Line ${x + 1}: ${lines[x]} should be of the form 'm=mediaType udpPort RTP/AVP payloadType'`));
+        continue;
+      }
```

Comment:
Could now do this in terms of `mediaMatch`? But maybe not worth it for now?

Reply:
Made this change and committed